test/f_ether_dividends.js

```diff
@@ -264,43 +265,48 @@ contract("EtherDividendCheckpoint", async (accounts) => {
         it("Should fail in creating the dividend", async () => {
             let maturity = await latestTime();
             let expiry = await latestTime() + duration.days(10);
-            await catchRevert(I_EtherDividendCheckpoint.createDividend(maturity, expiry, dividendName, { from: token_owner }));
+            await catchRevert(I_EtherDividendCheckpoint.createDividend(maturity, expiry, dividendName, { from: token_owner }),
+                "No dividend sent");
         });
 
-        it("Should fail in creating the dividend", async () => {
+        it("Should fail in creating the dividend -- invalid maturity/expiry", async () => {
             let maturity = await latestTime();
             let expiry = await latestTime() - duration.days(10);
             await catchRevert(
                 I_EtherDividendCheckpoint.createDividend(maturity, expiry, dividendName, {
                     from: token_owner,
                     value: new BN(web3.utils.toWei("1.5", "ether"))
-                })
+                }),
+                "Expiry is before maturity"
             );
         });
 
-        it("Should fail in creating the dividend", async () => {
+        it("Should fail in creating the dividend - Expiry is in the past", async () => {
             let maturity = await latestTime() - duration.days(2);
             let expiry = await latestTime() - duration.days(1);
             await catchRevert(
                 I_EtherDividendCheckpoint.createDividend(maturity, expiry, dividendName, {
                     from: token_owner,
                     value: new BN(web3.utils.toWei("1.5", "ether"))
-                })
+                }),
+                "Expiry is in the past"
             );
         });
 
         it("Set withholding tax of 20% on investor 2", async () => {
             await I_EtherDividendCheckpoint.setWithholdingFixed([account_investor2], new BN(web3.utils.toWei("0.2", "ether")), { from: token_owner });
         });
 
+        // @FIXME why does this transaction revert?
```

Comment:
`require(_name[0] != bytes32(0));`

Name shouldn't be empty. i.e 0x0, 0x00, 0x000..... are not allowed. Should edit test description to reflect this.